bot/exts/utils/utils.py

```diff
@@ -156,18 +157,22 @@ async def zen(self, ctx: Context, *, search_value: Union[int, str, None] = None)
 
     @command(aliases=("snf", "snfl", "sf"))
     @in_whitelist(channels=(Channels.bot_commands,), roles=STAFF_ROLES)
-    async def snowflake(self, ctx: Context, snowflake: Snowflake) -> None:
+    async def snowflake(self, ctx: Context, *snowflakes: Snowflake) -> None:
         """Get Discord snowflake creation time."""
-        created_at = snowflake_time(snowflake)
-        embed = Embed(
-            description=f"**Created at {created_at}** ({time_since(created_at, max_units=3)}).",
-            colour=Colour.blue()
-        )
-        embed.set_author(
-            name=f"Snowflake: {snowflake}",
-            icon_url="https://github.com/twitter/twemoji/blob/master/assets/72x72/2744.png?raw=true"
-        )
-        await ctx.send(embed=embed)
+        if len(snowflakes) > 1 and await has_no_roles_check(ctx, *STAFF_ROLES):
+            raise BadArgument("Cannot process more than one snowflake in one invocation.")
```

Comment:
Please mention here that this is only case if user is not staff member.

Reply:
That's probably not necessary. We can let the staff know that or alternatively add it to the help section of the command.

Reply:
I don’t think it is necessary, we don’t mention that only moderators can use `!user` on other users than themselves. I don’t mind adding it to the docstring but I feel like that will just add noise since the new command signature will reflect this change, what do you think?